lib/modules/platform/types.ts

```diff
@@ -288,7 +288,18 @@ export interface Platform {
     branchName: string,
     internalChecksAsSuccess: boolean,
   ): Promise<BranchStatus>;
-  getBranchPr(branchName: string, targetBranch?: string): Promise<Pr | null>;
+  getBranchPr(
+    branchName: string,
+    /**
```

Comment:
Also, mention that it is only being used for azure as of now. So it's displayed when someone tries to use it for another platform.